src/pages/en/guides/migrate-to-astro/from-gatsby.mdx

```diff
@@ -2,67 +2,884 @@
-    ```
-    </Fragment>
-    <Fragment slot="pnpm">
-    ```shell
-    pnpm create astro --template withastro/blog
-    ```
-    </Fragment>
-    <Fragment slot="yarn">
-    ```shell
-    yarn create astro --template withastro/blog
+- Option 1: **Create a new Astro project** 
+
+  Use `npm create astro@latest` to launch Astro's CLI wizard or choose a community theme from the [Astro Theme Showcase](https://astro.build/themes). 
+  
+  
+  You can also pass a `--template` argument to the `create-astro` command to start from an existing Astro repository on GitHub:
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      npm create astro -- --template <github-username>/<github-repo>
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm create astro --template <github-username>/<github-repo>
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn create astro --template <github-username>/<github-repo>
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+  
+  Then, copy your existing Gatsby project files over to your new Astro project into a separate folder outside of `src`.
+
+- Option 2: **Update your existing Gatsby project dependencies**
+
+  Remove any Gatsby-related dependencies in `package.json` and follow all the steps to [install Astro manually](/en/install/manual/) in your existing repository. (You may wish to do this on a different branch.) 
+  
+  Make sure you have [these project files](/en/install/manual/#7-next-steps) and a working home page before continuing. 
+
+:::tip
+Prefer to explore in your browser instead? Visit https://astro.new for a variety of starter templates that can be opened and edited in StackBlitz, CodeSandbox or Gitpod.
+:::
+
+### Install integrations (optional)
+
+You may find it useful to install some of [Astro's optional integrations](/en/guides/integrations-guide/) to use while converting your Gatsby project to Astro:
+
+- **@astrojs/react**: to reuse some existing React UI components in your new Astro site, or keep writing with React components.
+
+- **@astrojs/image**: to replace Gatsby's Image plugin with Astro's own image-optimizing components.
+
+- **@astrojs/mdx**: to bring existing MDX files from your Gatsby project, or to use MDX in your new Astro site.
+
+### Put your code in `src`
+
+1. **Delete** Gatsby's `public/` folder. 
+    
+    Gatsby uses the `public/` directory for its build output, so you can safely discard this folder. You will no longer need a built version of your Gatsby site. (Astro uses `dist/` by default for the build output.)
+
+2. **Rename** Gatsby's `static/` folder to `public/`, and use it as Astro's `public/` folder. 
+
+    Astro uses a folder called `public/` for static assets. You can alternatively copy the contents of `static/` into an existing Astro `public/` folder.
+
+3. **Copy or Move** Gatsby's other files and folders (e.g. `components`, `pages` etc.) into Astro's `src/` folder.
```

Comment:
For my project, this did not work. Once I move a lot of files in VS Code, all manner of imports and such get updated and the system tries to build all those files. I did get many errors all over the place – too many to fix one by one.

Instead I moved on page at a time and got that page working. This way the errors are managable.

There is also quite a bit of stuff that needs updating, like Gatsby Types (if using TS), imports that are Gatsby specific, like changing all `<Link to…`<a href…` and such.